picard/file.py

```diff
@@ -683,13 +682,19 @@ def _tags_to_update(self, ignored_tags):
         for name in set(self.metadata) | set(self.orig_metadata):
             if name.startswith('~'):
                 continue
-            if not self.supports_tag(name):
-                continue
             if name in ignored_tags:
                 continue
+            if not self.supports_tag(name):
+                continue
             yield name
 
     def update(self, signal=True):
+        thread.run_task(
+            self._compare_metadata,
+            partial(self._compare_metadata_finished, signal)
+        )
```

Comment:
Is using a separate (unspecified) thread pool here going to cause some commands to show as being completed prematurely in the testing in `picard/tagger.py` Lines 404 - 434?

Reply:
We need to be careful that we do not run more than one CPU intensive thread at a time. We should have a separate thread pool for CPU intensive workload with a single thread in it.

Reply:
@rdswift As I understand it, no. These updates ran after file loading anyway, when the file gets moved to a cluster or release.

@Sophist-UK I moved the `file.update` to use `tagger.priority_thread_pool` instead (previously only used by the metadata view). That is limited to one thread. And I think it also fits with the purpose. Essentially these are updates that are supposed to update the UI, and they should happen in time.

This also solves the issue that these updates must not be put behind all the loading threads, otherwise you just end with a longer UI stutter once all loading has ended. I first attempted to solve this by setting higher thread pool priority for `file.update`, but that lead to much more stuttering.  Having these update calls be run sequentially indeed improved responsiveness for me again.

Reply:
It definitely shouldn't be a problem now that you're running the task in the `tagger.priority_thread_pool`.  Thanks.